data-prepper-plugins/saas-source-plugins/jira-source/src/main/java/org/opensearch/dataprepper/plugins/source/jira/utils/Constants.java

```diff
@@ -0,0 +1,63 @@
+package org.opensearch.dataprepper.plugins.source.jira.utils;
+
+import java.util.List;
+
+/**
+ * The type Constants.
+ */
+public class Constants {
```

Comment:
Rather than a big `Constants` class move constants into the classes that need them. If they are needed in multiple places, create smaller classes that encapsulate concepts. For example, `RestConstants`.

Reply:
Done. Introduced a new class and moved some of the constants